Zip with keywords arguments not supported in python 3.9

### 🐛 Describe the bug

On python 3.9 I get:

```

in zip(
            _chosen_input_chunks,
            _rejected_input_chunks,
            _chosen_target_chunks,
            _rejected_target_chunks,
            (_ref_chosen_input_chunks if use_ref_model else [None] * len(_chosen_input_chunks)),
            (_ref_rejected_input_chunks if use_ref_model else [None] * len(_rejected_input_chunks)),
            (_chosen_nll_target_chunks if nll_target is not None else [None] * len(_chosen_input_chunks)),
            strict=False,
        ):
E       TypeError: zip() takes no keyword arguments

.venv/lib/python3.9/site-packages/liger_kernel/chunked_loss/fused_linear_preference.py:217: TypeError
```


### Reproduce

_No response_

### Versions

Liger Kernel version: 0.5.9

--- a/benchmark/benchmarks_visualizer.py
+++ b/benchmark/benchmarks_visualizer.py
@@ -123,7 +123,7 @@ def plot_data(df: pd.DataFrame, config: VisualizationsConfig):
     lines = ax.get_lines()
     colors = [line.get_color() for line in lines]
 
-    for (_, group_data), color in zip(df.groupby("kernel_provider"), colors, strict=False):
+    for (_, group_data), color in zip(df.groupby("kernel_provider"), colors):
         # for i, row in group_data.iterrows():
         y_error_lower = group_data["y_value_50"] - group_data["y_value_20"]
         y_error_upper = group_data["y_value_80"] - group_data["y_value_50"]

--- a/examples/huggingface/training_multimodal.py
+++ b/examples/huggingface/training_multimodal.py
@@ -68,7 +68,7 @@ def construct_model_and_processor(model_name: str, use_liger: bool) -> torch.nn.
 def _validate_and_extract_the_cauldron(examples) -> dict[str, list]:
     batch_texts = []
     batch_images = []
-    for images, texts in zip(examples["images"], examples["texts"], strict=False):
+    for images, texts in zip(examples["images"], examples["texts"]):
         if not images:
             raise ValueError("No image found in example from the_cauldron dataset")
         if len(images) > 1:

--- a/examples/medusa/train.py
+++ b/examples/medusa/train.py
@@ -175,7 +175,7 @@ def preprocess(
     input_ids = encoding.input_ids
 
     # Mask targets. Only compute loss on the assistant outputs.
-    for conv_index, (conversation, target, prompt) in enumerate(zip(conversations, targets, prompts, strict=False)):
+    for conv_index, (conversation, target, prompt) in enumerate(zip(conversations, targets, prompts)):
         # print(conv_index)
         for turn in conversation:
             if turn["role"] == "assistant":

--- a/src/liger_kernel/chunked_loss/fused_linear_preference.py
+++ b/src/liger_kernel/chunked_loss/fused_linear_preference.py
@@ -222,7 +222,6 @@ class LigerFusedLinearPreferenceBase(torch.autograd.Function):
             (_ref_chosen_input_chunks if use_ref_model else [None] * len(_chosen_input_chunks)),
             (_ref_rejected_input_chunks if use_ref_model else [None] * len(_rejected_input_chunks)),
             (_chosen_nll_target_chunks if nll_target is not None else [None] * len(_chosen_input_chunks)),
-            strict=False,
         ):
             input_chunk = torch.cat([chosen_input_chunk, rejected_input_chunk], dim=0)
             ref_input_chunk = (

--- a/test/convergence/bf16/test_mini_models.py
+++ b/test/convergence/bf16/test_mini_models.py
@@ -1130,6 +1130,5 @@ def test_mini_model(
     for expected_param, actual_param in zip(
         expected_output["model"].named_parameters(),
         actual_output["model"].named_parameters(),
-        strict=False,
     ):
         assert_verbose_allclose(expected_param[1], actual_param[1], atol=param_atol, rtol=param_rtol)

--- a/test/convergence/bf16/test_mini_models_multimodal.py
+++ b/test/convergence/bf16/test_mini_models_multimodal.py
@@ -929,6 +929,5 @@ def test_mini_model_multimodal(
     for expected_param, actual_param in zip(
         expected_output["model"].named_parameters(),
         actual_output["model"].named_parameters(),
-        strict=False,
     ):
         assert_verbose_allclose(expected_param[1], actual_param[1], atol=param_atol, rtol=param_rtol)

--- a/test/convergence/bf16/test_mini_models_with_logits.py
+++ b/test/convergence/bf16/test_mini_models_with_logits.py
@@ -1129,6 +1129,5 @@ def test_mini_model(
     for expected_param, actual_param in zip(
         expected_output["model"].named_parameters(),
         actual_output["model"].named_parameters(),
-        strict=False,
     ):
         assert_verbose_allclose(expected_param[1], actual_param[1], atol=param_atol, rtol=param_rtol)

--- a/test/convergence/fp32/test_mini_models.py
+++ b/test/convergence/fp32/test_mini_models.py
@@ -1000,6 +1000,5 @@ def test_mini_model(
     for expected_param, actual_param in zip(
         expected_output["model"].named_parameters(),
         actual_output["model"].named_parameters(),
-        strict=False,
     ):
         assert_verbose_allclose(expected_param[1], actual_param[1], atol=param_atol, rtol=param_rtol)

--- a/test/convergence/fp32/test_mini_models_multimodal.py
+++ b/test/convergence/fp32/test_mini_models_multimodal.py
@@ -910,6 +910,5 @@ def test_mini_model_multimodal(
     for expected_param, actual_param in zip(
         expected_output["model"].named_parameters(),
         actual_output["model"].named_parameters(),
-        strict=False,
     ):
         assert_verbose_allclose(expected_param[1], actual_param[1], atol=param_atol, rtol=param_rtol)

--- a/test/convergence/fp32/test_mini_models_with_logits.py
+++ b/test/convergence/fp32/test_mini_models_with_logits.py
@@ -1000,6 +1000,5 @@ def test_mini_model(
     for expected_param, actual_param in zip(
         expected_output["model"].named_parameters(),
         actual_output["model"].named_parameters(),
-        strict=False,
     ):
         assert_verbose_allclose(expected_param[1], actual_param[1], atol=param_atol, rtol=param_rtol)

